tikv/tikv
# status_server: add pprof flamegraph response header

Signed-off-by: Suhaha <jklopsdfw@gmail.com>



### What problem does this PR solve?

Problem Summary:

No related PR. The response of svg returned by the status server's pprof does not have a corresponding Content-Type header. And the correct return header makes it easier for different clients to handle the mime type.

### What is changed and how it works?

What's Changed:

Add the correct Content-Type to response header. 

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test
```
curl -v http://127.0.0.1:20180/debug/pprof/profile\?
```
will see the response header with Content-Type
```
< HTTP/1.1 200 OK
< content-type: image/svg+xml
< content-length: 187695
< date: Wed, 15 Sep 2021 15:49:13 GMT
```

Side effects

### Release note 

```release-note
Add the correct Content-Type to `debug/pprof/profile` response header. 
```
